sites/footers/includes/footer.html

```diff
@@ -1,29 +1,84 @@
 <footer id="wb-info">
+{% unless page.noFooterContextual %}
+{%- if site.global.contextualFooter.title[i18nText-lang] and site.global.contextualFooter.links[i18nText-lang] -%}
+	<div class="gc-contextual">
+		<nav class="container">
+			{%- if page.contextualFooter.title -%}
+			<h3>{{ page.contextualFooter.title }}</h3>
+			{%- else -%}
+			<h3>{{ site.global.contextualFooter.title[i18nText-lang] }}</h3>
+			{%- endif -%}
+			<ul class="list-col-xs-1 list-col-sm-2 list-col-md-3">
+				{%- if page.contextualFooter.links -%}
+				{%- for link in page.contextualFooter.links -%}
+				<li><a href="{{ link.url }}">{{ link.text }}</a></li>
+				{%- endfor -%}
+				{%- else -%}
+				{%- for link in site.global.contextualFooter.links[i18nText-lang] -%}
+				<li><a href="{{ link.url }}">{{ link.text }}</a></li>
+				{%- endfor -%}
+				{%- endif -%}
+			</ul>
+		</nav>
+	</div>
+{%- endif -%}
+{% endunless %}
+
 {% unless page.noFooterAbout %}
 	<div class="landscape">
-		<nav class="container wb-navcurr">
-			<h2 class="wb-inv">{{ i18nText-footerAbout }}</h2>
-			<ul class="list-unstyled colcount-xs-2 colcount-md-3">
-{%- if i18nText-lang == "fr" -%}
-				<li><a href="https://www.canada.ca/fr/contact.html">Contactez-nous</a></li>
+		<nav class="container">
+			<h3>{{ i18nText-goc }}</h3>
```

Comment:
What about "Support links" and keep "About this site" for the "<footer>" element.

UK are using "Support links" for similar content.

@delisma what do you think?

Reply:
@pcwsmith @LPipes4 I know this header is hidden but we're are calling this section "Sub-footer links", do we want to align our documentation in order to have devs and comms speaking the same language?

Reply:
@duboisp Talked to @pcwsmith and @LPipes4 and it seems that we will align with you terminology but we have some reserved on using Support because it's still super broad and vague.